src/NuGet.Core/NuGet.Protocol/Plugins/Messages/GetAuthenticationCredentialsRequest.cs

```diff
@@ -37,11 +43,12 @@ public sealed class GetAuthenticationCredentialsRequest
         /// <param name="isNonInteractive"></param>
         /// <exception cref="ArgumentNullException"> if <paramref name="uri"/> is null</exception>
         [JsonConstructor]
-        public GetAuthenticationCredentialsRequest(Uri uri, bool isRetry, bool isNonInteractive)
+        public GetAuthenticationCredentialsRequest(Uri uri, bool isRetry, bool isNonInteractive, bool canPrompt)
```

Comment:
just for my understanding, what is difference between IsNonInteractive and canPrompt? wasn't interactive flag was also to allow user to input credentials?